src/logger/logger-wrapper.ts

```diff
@@ -39,11 +39,13 @@ function isInitialized(): boolean {
  *
  * @returns { IVSCodeExtLogger }
  */
-export function getLogger(): IVSCodeExtLogger | undefined {
+export function getLogger(): IVSCodeExtLogger {
```

Comment:
Now, you should change all the places that use `?.` after calling `getLogger()` and the types of variables assigned from `getLogger()` not to include `undefined`, if their types are declared. This includes the code you changed in `extension.ts` to check if `logger` is undefined...

Reply:
Done